app/assets/stylesheets/grader.scss

```diff
@@ -51,32 +51,32 @@
   width: 100%;
 
   li {
-    border-bottom: 1px solid $light-grey;
+    border-bottom: 1px solid $background-main;
     padding: 5px;
     width: 100%;
     display: inline-block;
   }
 
   .unassigned {
-    background: $light-grey;
-    color: $grey;
+    background: $background-main;
```

Comment:
This one is supposed to be faded to indicate that it's read-only.

Reply:
Is this:
<img width="431" alt="image" src="https://user-images.githubusercontent.com/29822176/98283082-1e966080-1f6d-11eb-97a6-7988eff0e10e.png">
more in line with what the disabled marks should look like?

Reply:
I like that more, although I would prefer that the "selected level" colour be different than the "disabled" colour